timeless/items/views.py

```diff
@@ -33,4 +42,4 @@ def edit():
 @BP.route("/delete", methods=["POST"])
 def delete():
     """ Delete an item by id """
-    return redirect(url_for("items.list_items"))
+    return render_template("items/list.html")
```

Comment:
@paulodamaso Why did you decide to change redirect to `render_template` ?

Reply:
@vladarefiev redirect failed with the change to `ListView`

Reply:
@paulodamaso pls leave a puzzle to fix it. because rendering list.html is wrong logic